frontend/src/components/PlannerChart.module.scss

```diff
@@ -19,4 +21,38 @@
 
 .label {
   @extend .typography-pre-title;
+  fill: $COLOR_BLACK60;
+}
+
+:global(.recharts-cartesian-axis-tick-value) {
+  @extend .typography-pre-title;
+  fill: $COLOR_BLACK100;
+  letter-spacing: 0;
+}
+
+.title {
+  @extend .typography-body-bold;
+  display: flex;
+  flex-direction: row;
```

Comment:
This could extend to the class `.layout-row` instead as it has been a default style in this project